contrib/guix/libexec/build.sh

```diff
@@ -1,4 +1,5 @@
 #!/usr/bin/env bash
+export TZ=UTC
```

Comment:
in commit 5f8a2f43ff3becd329da9c9b303fb4322ad71a08:

Can this be set after LC_ALL=C to make travis pass?